ee/modules/600-flant-integration/images/madison-proxy/madison-proxy.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2021 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"net/http"
+	"net/http/httputil"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const logFormat = `request:%s %s %s, status:%s, body:%s`
```

Comment:
Previously, the proxy logs were in JSON format. Logrus natively supports JSON.

Anyway, let's leave logs recording as it is right now in this PR. It is not a big deal.

Reply:
now log.SetFormatter is set to log.JSONFormatter{}